src/libraries/System.Net.Quic/src/System/Net/Quic/Implementations/MsQuic/MsQuicListener.cs

```diff
@@ -120,6 +120,13 @@ internal override void Start()
         {
             ThrowIfDisposed();
 
+            // protect against double starts.
+            if (_started)
```

Comment:
Why is Start() being called multiple times? Is it a misusage where it would be better to throw?

Reply:
Yeah misusage somewhere, wasn't able to find exactly. Do you think it's better to throw here?

Reply:
I think throwing is probably better/easier. Otherwise you run into issues like deciding whether to let the second call to Start() complete before the first one.